### PR TITLE
Update libraries.md to be more detailed

### PR DESCRIPTION
**Pull request by CouldBeThis**
I made instructions for adding libraries more explicit because I got lost in the menus myself. 

pictures/animation would be better but I'm not sure how to upload screen caps and also I don't know if that would be useful/welcome.

### CODE REVIEW

**Comment by dkanada** [comment]
We're fine with images, and they can be added in the `images` directory and referenced in the markdown files. Here are the linting errors for this pull request, let me know if any of them are unclear.

```
general/server/libraries.md:12:91 MD034/no-bare-urls Bare URL used [Context: "http://localhost:8096/"]
general/server/libraries.md:13:206 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
general/server/libraries.md:14:30 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
general/server/libraries.md:15:57 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
general/server/libraries.md:17:252 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
general/server/libraries.md:18:302 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```

**Comment by CouldBeThis** [comment]
"markdown linting" what a neat concept, I am going to look at it further. 

I *think* I have made the edits requested and put the file in the correct place. If not I will be grateful to know about it so I can do it properly next time. 

thank you for your patience

File: general/server/libraries.md [hunk 1]
```diff
@@ -5,7 +5,19 @@ title: Libraries
...
 # Libraries
 
-You will see a page to add libraries when you first create the server, but they can also be added or removed at any time from the settings. Libraries are virtual collections of media and can contain files from several different locations on the server.
+Libraries are virtual collections of media and can contain files from several different locations on the server.
+
+You will see a page to add libraries when you first create the server, but they can also be added or removed at any time from the settings:
+
+1. Log in to Jellyfin web interface in your web browser. The default location for this is <http://localhost:8096/>.
+2. Click the hamburger menu (<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Hamburger_icon.svg/191px-Hamburger_icon.svg.png" width="24"/>) in the top left hand corner to open the menu.
+3. Select `User` > `Settings`
```
Comment (Artiume) [inline general/server/libraries.md:14]: I don't think this step is necessary

File: general/server/libraries.md [hunk 2]
```diff
@@ -5,7 +5,19 @@ title: Libraries
...
+4. On the menu that appears, click `Admin` > `Dashboard`
```
Comment (Artiume) [inline general/server/libraries.md:15]:
```suggestion
4. On the menu that appears, click `Admin` > `Dashboard`.
```

File: general/server/libraries.md [hunk 3]
```diff
@@ -5,7 +5,19 @@ title: Libraries
...
+5. Then again on the left side menu go to `Server` > `Libraries`
```
Comment (Artiume) [inline general/server/libraries.md:16]:
```suggestion
5. Then again on the left side menu go to `Server` > `Libraries`.
```

File: general/server/libraries.md [hunk 4]
```diff
@@ -5,7 +5,19 @@ title: Libraries
...
+6. In the main/middle part of the screen you will see the categories of media you have already added. The last item will be "Add Media Library". Selecting that item will initiate the process just like in the installation, just follow the instructions.
```
Comment (Artiume) [inline general/server/libraries.md:17]: The first part of this sentence is overly detailed, while the latter half is overly simplified. 
```suggestion
6. Click "Add Media Library". 
```

File: general/server/libraries.md [hunk 5]
```diff
@@ -5,7 +5,19 @@ title: Libraries
...
+7. When you are done you will be returned to the page from Step 5. The server will need time to catalogue your newly-added files. There will be a progress bar at the top of the page indicating how far along it is. If you add a large amount of media at once, it could take a while so please be patient.
```
Comment (Artiume) [inline general/server/libraries.md:18]:
```suggestion
7. The server will now add your new media. There will be a progress bar at the top of the page indicating its progress.
```

File: general/server/libraries.md [hunk 6]
```diff
@@ -5,7 +5,19 @@ title: Libraries
...
+
+<!-- if any troubleshooting infos are available, provide links -->
```
Comment (Artiume) [inline general/server/libraries.md:20]: what's this for?

Comment (dkanada) [inline general/server/libraries.md:20]:
```suggestion
```

File: general/server/libraries.md [hunk 7]
```diff
@@ -5,7 +5,17 @@ title: Libraries
...
+Libraries are virtual collections of media and can contain files from several different locations on the server.
+
+You will see a page to add libraries when you first create the server, but they can also be added or removed at any time from the settings:
+
+1. Log in to Jellyfin web interface in your web browser. The default location for this is <http://localhost:8096/>.
+2. Click the hamburger menu (<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Hamburger_icon.svg/191px-Hamburger_icon.svg.png" width="24"/>) in the top left hand corner to open the menu.
+3. Select `User` > `Settings`
+4. On the menu that appears, click `Admin` > `Dashboard`.
+5. Then again on the left side menu go to `Server` > `Libraries`.
+6. Click "Add Media Library".
```
Comment (dkanada) [inline general/server/libraries.md:17]:
```suggestion
6. Click "Add Media Library".
```

**Review by dkanada — APPROVED** [review]

**Pull request merged**